framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1338,33 +1176,23 @@ wifi_manager_result_e _handler_on_softap_state(_wifimgr_msg_s *msg)
 		WIFIMGR_CHECK_RESULT(_wifimgr_scan((wifi_manager_ap_config_s *)msg->param), "fail scan", WIFI_MANAGER_FAIL);
 		WIFIMGR_STORE_PREV_STATE;
 		WIFIMGR_SET_STATE(WIFIMGR_SCANNING);
+#ifdef CONFIG_WIFIMGR_DISABLE_DHCPS
```

Comment:
ifdef is correct? or ifndef is correct..?

Reply:
Here, ifdef is correct. If DHCPS is disabled then EVN_JOINED will be used to manage the join callback and status. Thanks for the comment!